### ISSUE TITLE
Support configurations of global check constants

### ISSUE THREAD

**Issue by niksawtschuk** [google/GTXiLib#8]
As mentioned in #6, it would be a good idea to be able to override global check constants, like tappable area and contrast ratio.

For example, `kGTXMinContrastRatioForAccessibleText` is set to 3.0. In [Apple's guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/), it is recommended to maintain a minimum contrast ratio of 4.5:1. In the [WCAG rules](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) for contrast, a ratio of 4.5:1 must be maintained for all text except for sufficiently large text or logos.

If it was possible to override these constants to fit your need, the library would be more flexible. It also calls into question how large text should be handled, as different ratios may be required.

Different WCAG levels also require different ratios, so this may be yet-another level of configuration we could support.

**Comment by j-sid** [comment]
This is the direction we are taking as well, we plan to provide a config object that can be provided to GTX for the checks to read from and configure themselves. Please let us know if this would not work for you or have a better idea.

**Comment by j-sid** [comment]
We will start with configs for contrast ratio and touch target size, and plan to expand it for more checks.

**Comment by niksawtschuk** [comment]
@j-sid that sounds great. I think it might be useful to have configuration "defaults" that align with a11y standards around the world. For example, there could be an easy way to quickly set the tool to use WCAG AA or AAA rules. I think this may also help clear up #1

**Comment by j-sid** [comment]
Cool, I like the idea of using AA or AAA configs I just want to ensure that we make it clear that passing AA/AAA on GTX does not mean the app is A11y ready, only means that all AA/AAA checks in GTX have passed.
